### PR TITLE
fr: fix url locales

### PR DESCRIPTION
**Pull request by yin1999**
### Description

fix url locales, using the script in #15638

### CODE REVIEW

File: files/fr/web/javascript/reference/global_objects/error/stack/index.md [hunk 1]
```diff
@@ -112,7 +112,6 @@ Ne fait partie d'aucune spécification. Non-standard.
 
 ## Voir aussi
 
-- [Components.stack](/fr-FR/docs/Components.stack)
```
Comment (yin1999) [inline files/fr/web/javascript/reference/global_objects/error/stack/index.md:115]: removed, as this is not existed in en-US

File: files/fr/web/api/window/index.md [hunk 2]
```diff
@@ -82,8 +82,6 @@ Notez que les propriétés qui sont des objets (par exemple, pour redéfinir le
   - : Renvoie une référence à l'objet de stockage local utilisé pour stocker les données accessibles uniquement par l'origine qui les a créées.
 - {{domxref("Window.menubar")}} {{ReadOnlyInline}}
   - : Renvoie l'objet barre de menus, dont la visibilité peut être inversée dans la fenêtre.
-- {{domxref("Window.messageManager")}}
-  - : Renvoie l'objet [gestionnaire de messages](/fr-FR/docs/The_message_manager) pour cette fenêtre.
```
Comment (yin1999) [inline files/fr/web/api/window/index.md:86]: removed from en-US

File: files/fr/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Maintenant que notre programme peut faire tourner un cube 3D, appliquons lui une
 
 La première chose à faire est d'ajouter le code pour charger les textures. Dans notre cas, nous utiliserons une texture unique, appliquée à chacune des six faces de notre cube en rotation ; mais la même technique peut être utilisée un nombre quelconque de textures.
 
-> **Note :** il est important de noter que le chargement des textures suit les [règles inter-domaines](/fr-FR/docs/Web/HTTP/CORS)&nbsp;; donc vous pouvez seulement charger des textures depuis les sites pour lesquels votre contenu a l'approbation CORS. Voir les textures inter-domaines ci-dessous pour plus de détails.
```
Comment (yin1999) [inline files/fr/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md:14]: I'm not sure could I remove the space `&nbsp;;`, if could not, feel free tor restore it.

**Review by SphinxKnight — APPROVED** [review]
wow, thanks a lot @yin1999 !

**Pull request merged**